docs/src/utils/track.ts

```diff
@@ -0,0 +1,161 @@
+// https://github.com/aws-amplify/docs/blob/main/src/utils/track.ts
+let configured = false;
+
+declare global {
+  interface Window {
+    s: any;
```

Comment:
can we use a different variable name? `s` seems to be a little bit confusing

Reply:
Pretty sure `s` is what Adobe analytics requires...

Reply:
Yup that variable is not set by us